.env.local-example

```diff
@@ -27,4 +27,6 @@ PLAYWRIGHT_PRODUCT_SIZE_OUTOFSTOCK='XS'
 PLAYWRIGHT_PRODUCT_COLOR_OUTOFSTOCK='Gray'
 PLAYWRIGHT_COLLECTION_NAME='Men'
 PLAYWRIGHT_BRAND_NAME='Nike'
+PLAYWRIGHT_USER_EMAIL=''
```

Comment:
The empty strings are the right way to go here. 

I'd suggest editing all the `PLAYWRIGHT_` vars in `.env.local-example` to make them empty strings. Most people setting up a project and running this will have a `Nike` brand or a `Men` collection. This is user input.

Reply:
done